TYPO3.Flow/Classes/TYPO3/Flow/Security/Account.php

```diff
@@ -347,6 +360,51 @@ public function setExpirationDate(\DateTime $expirationDate = null)
     }
 
     /**
+     * @return integer
+     * @api
+     */
+    public function getFailedAuthenticationCount()
+    {
+        return $this->failedAuthenticationCount;
+    }
+
+    /**
+     * @param integer $failedAuthenticationCount
+     * @return void
+     */
+    public function setFailedAuthenticationCount($failedAuthenticationCount)
+    {
+        $this->failedAuthenticationCount = $failedAuthenticationCount;
+    }
```

Comment:
There is already a `setAuthenticationStatus` method, which is called from the `PersistedUsernamePasswordProvider`. Would you set / reset the two values in this setter as well? Or would you introduce two new methods authenticationSuccessful() and authenticationFailed()?